app-backend/migrations/src_migrations/main/scala/123.scala

```diff
@@ -0,0 +1,28 @@
+import slick.jdbc.PostgresProfile.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M123 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(123)(List(
+    sqlu"""
+    ALTER TABLE platforms
+      RENAME COLUMN settings
+      TO public_settings;
+
+    ALTER TABLE platforms
+      ADD COLUMN private_settings JSONB NOT NULL default '{}';
```

Comment:
This default is different from what gets inserted in the generators -- what happens if I try to get a platform with empty json here? Should the default match the value set in the `UPDATE` below? I don't think that's tested anywhere:

- platforms already in the database get their records set to json with all the keys and empty string values in the migration
- platforms inserted in testing have fully filled in public and private settings

I think the better solution is just to default to the value that you're setting as the default for existing platforms instead of `{}`. But if you wanted to, `().asJson` would generate an empty json for testing.